packages/multi-select-combo-box/test/validation.test.js

```diff
@@ -0,0 +1,45 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import './not-animated-styles.js';
+import '../vaadin-multi-select-combo-box.js';
+
+describe('validation', () => {
+  let comboBox, validateSpy;
```

Comment:
Do we need `validateSpy` also for any other test suite? If not, let's move it to `basic` suite for now.

Reply:
Now it is also used in the `initial` suite.